cmd/parca-agent/main.go

```diff
@@ -240,8 +243,8 @@ type FlagsHidden struct {
 	AllowRunningAsNonRoot             bool `help:"Force running the Agent even if the user is not root. This will break a lot of the assumptions and result in the Agent malfunctioning."  hidden:""`
 	AllowRunningInNonRootPIDNamespace bool `help:"Force running the Agent in a non 'root' PID namespace. This will break a lot of the assumptions and result in the Agent malfunctioning." hidden:""`
 
-	EnablePythonUnwinding bool `default:"false" help:"Enable Python unwinding." hidden:""`
-	EnableRubyUnwinding   bool `default:"false" help:"Enable Ruby unwinding."   hidden:""`
+	EnablePythonUnwinding bool `default:"false" help:"[deprecated] Python unwinder enabled by default. Use --python-unwinding-disable to disable it." hidden:""`
+	EnableRubyUnwinding   bool `default:"false" help:"[deprecated] Ruby unwinder enabled by default. Use --ruby-unwinding-disable to disable it."     hidden:""`
```

Comment:
Happy it you prefer to remove it now and add it to the release notes as a backwards incompatible change, your call!

Reply:
We can add it to the changelog and we can remove it in subsequent releases.

Reply:
Completely up to you!